pandas/tests/series/test_missing.py

```diff
@@ -410,6 +410,17 @@ def test_isna_for_inf(self):
         tm.assert_series_equal(r, e)
         tm.assert_series_equal(dr, de)
 
+    def test_isnull_for_inf_deprecated(self):
+        # gh-17115
+        s = Series(['a', np.inf, np.nan, 1.0])
+        with pd.option_context('mode.use_inf_as_null', True):
```

Comment:
It seems strange that this doesn't show up? Doesn't that mean it is triggered somewhere else (which shouldn't be the case, because we should never use this option somewhere else?)